_posts/0000-01-02-Sebabats.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Sebabats will add a new slide
```

Comment:
When life gives you lemons, make strawberry juice and leave them wondering how you did it